src/core/src/bootstrap/Constants.py

```diff
@@ -29,6 +29,9 @@ def __iter__(self):
     GLOBAL_EXCLUSION_LIST = ""   # if a package needs to be blocked across all of Azure
     UNKNOWN = "Unknown"
 
+    # Extension version (todo: move to a different file)
+    EXT_VERSION = "1.6.15"
```

Comment:
Please create backlog task and link task here to close this comment.

Reply:
https://msazure.visualstudio.com/One/_queries/edit/9388858/?triage=true